source/api_reference/search_payments_reference/index.html.md.erb

```diff
@@ -194,14 +194,15 @@ After you send your request to search payments, you’ll receive a response simi
+| `card_details.wallet_type` | string | The digital wallet the user paid with.<br><br>Possible values are:<br><br><li>`Apple Pay`<li>`Google Pay`<br><br>`wallet_type` only appears if the user paid with a digital wallet. You can [read more about taking digital wallet payments](/digital_wallets/).                                                                                                                                                            |
 | `delayed_capture` | boolean | `delayed_capture` is `true` if you're controlling how long it takes GOV.UK Pay to take ('capture') a payment.<br><br>You can [read more about delaying taking a payment](/delayed_capture). |
 | `moto` | boolean | Indicates if this payment is a Mail Order / Telephone Order (MOTO) payment.<br><br>You can [read more about taking payments over the phone](/moto_payments). |
 | `total_amount` | number | Amount your user paid in pence, including corporate card fees.<br><br>`total_amount` only appears if you added a corporate card surcharge to the payment. <br><br>You can [read more about corporate card surcharges.](/corporate_card_surcharges/#add-corporate-card-fees). |
 | `fee` | number | The payment service provider's transaction fee, in pence.<br><br>`fee` only appears when we have taken ('captured') the payment from the user or if their payment fails after they submitted their card details.<br><br>You can [read more about PSP fees](/reporting/#psp-fees). |
 | `net_amount` | number | The amount, in pence, that'll be paid into your bank account after your payment service provider takes the `fee`. |
 | `provider_id` | string | The unique ID your payment service provider generated for this payment.<br><br>This is not the same as the `payment_id`. |
 | `return_url` | string | The URL you directed the paying user to after their payment journey on GOV.UK Pay ended.<br><br>You can [read more about choosing your return URL and matching the paying user to a payment](/making_payments/#choose-the-return-url-and-match-your-users-to-payments). |
-| `authorisation_mode` | string | How GOV.UK Pay will authorise this payment.<br><br>Possible values are:<br><br><li>`web`<li>`moto_api`<li>`agreement`<br><br>`authorisation_mode` is set when you create a payment. You can [read more about the different authorisation modes in our API reference page for creating payments](/api_reference/create_a_payment_reference). |
+| <nobr>`authorisation_mode` | string | How GOV.UK Pay will authorise this payment.<br><br>Possible values are:<br><br><li>`web`<li>`moto_api`<li>`agreement`<br><br>`authorisation_mode` is set when you create a payment. You can [read more about the different authorisation modes in our API reference page for creating payments](/api_reference/create_a_payment_reference). |
```

Comment:
There seems to be no closing `</nobr>` tag.

Also, should this indicate `web` is the default if it’s omitted?

Reply:
That [information exists in the reference page for `POST v1/payments`](https://docs.payments.service.gov.uk/api_reference/create_a_payment_reference/#json-body-parameters-for-39-create-a-payment-39) but it's left out here because this is just for searching for payments.